com.visualstudio.code-url-handler.desktop

```diff
@@ -0,0 +1,12 @@
+[Desktop Entry]
```

Comment:
Is it something that is also shipped in the official packages?

Reply:
Yes.